test/TestProjects/TenantOnly/TenantOnly.csproj

```diff
@@ -11,11 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.ResourceManager.Core" Version="1.0.0-alpha.20210706.2" />
+    <!-- <PackageReference Include="Azure.ResourceManager.Core" Version="1.0.0-alpha.20210706.2" /> -->
+    <ProjectReference Include="../../../../azure-sdk-for-net/sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj" />
```

Comment:
After https://github.com/Azure/azure-sdk-for-net/pull/22589 is merged and published, use the new version of core.